### ISSUE TITLE
Confused with remote data and path after login

### ISSUE THREAD

**Issue by ammichael** [nolanlawson/pouchdb-authentication#108]
Hi guys! 

First, I need to say I'm impressed with PouchDB+CouchDB. When I got it working for the first time, I felt a wizard myself. But the real magic has been done by you guys, so congrats!

The next step after making it work is to get it working right and play nice with security, so I need the file vault scenario, where each user has its own database, and can only read/write their own database.

I've managed to get the CouchPerUser plugin working, and whenever I signUp someone, it automatically creates a new database. But I'm not seeing how they are related. No metadata in user account linking with the new database or something like this.

I did try and use the response object the login method throws me back and sync with my local db, but all I 've got is this error:

`[Error] Failed to load resource: the server responded with a status of 400 (Bad Request) (HNhFCknLmYEBBaTA7sKU.w==, line 0)`

I did some few changes to keep it shorter, hope I've closed everything correctly.. This is the code: 

```
var dbLocal = new PouchDB('mydb');
var dbRemote = new PouchDB('http://127.0.0.1:5984', {skipSetup: true});
        
var login = function() {
	dbRemote.login('bruce', 'wayne').then(function() {
	return dbRemote;
	}).then(function(remote) {
		dbLocal.sync(remote, {
			live: true,
			retry: true
		}).on('change', function (change) {
			// yo, something changed!
			console.log('change')
			console.log(change)
		})
            })
	}
```

Is the remote path right? I thought, at first, that I would need to get the session data, which would have the name of the created database, and use it as part of the remote path..

What am I missing? 

Thank you!

**Comment by nolanlawson** [comment]
Everything in your code looks fine to me. Are you using an HTTP proxy? Which version of CouchDB/PouchDB are you using? Is this only in one browser, or in Node.js...?

Also you may find that the Slack channel or Google Groups are a better venue for this kind of discussion than GitHub Issues... there are lots of people in there and they usually respond fairly quickly.

**Comment by ammichael** [comment]
Actually, I've solved the problem the next day; but got other problems and wanted to solve all them before update here.

I've had problems to understand what was going on between PouchDB+pouchdb-authentication+CouchDB+CouchPerUser. At the end, it just needs to signUp/login the user, I'll get a response with the user name. Then I just convert the username do hex and prefix it with 'userdb-'. This is how CouchPerUser creates the db for an user.

Then, when I thought everything was fine, I've noticed problems with the login persistence. But it is Safari related.. I solved the issue storing using data in localstorage and doing login every run. Not perfect, but works pretty fine.

Thanks anyway!
